### PR TITLE
Added fun free courses for Ruby on Rails, jQuery, and SQL

### PR DESCRIPTION
**Pull request by tinabme**

### CODE REVIEW

File: free-courses-en.md [hunk 1]
```diff
@@ -137,6 +139,11 @@
 * [Problem Solving With Java](https://www.udacity.com/course/intro-to-java-programming--cs046)
 
 
+### jQuery
+
+* [CodeSchool - Try jQuery](https://www.codeschool.com/courses/try-jquery)
```
Comment (vhf) [inline free-courses-en.md]: Could you please move this link to `free-programming-interactive-tutorials-en.md`?

File: free-courses-en.md [hunk 2]
```diff
@@ -224,6 +232,11 @@
 * [Principles of Reactive Programming](https://www.coursera.org/course/reactive)
 
 
+### SQL
+
+* [Code School - Try SQL](https://www.codeschool.com/courses/try-sql)
```
Comment (vhf) [inline free-courses-en.md]: Could you please move this link to `free-programming-interactive-tutorials-en.md`?

File: free-courses-en.md [hunk 3]
```diff
@@ -210,6 +217,7 @@
 
 ### Ruby
 
+* [CodeSchool - Getting Started With Ruby on Rails](https://www.codeschool.com/learn/ruby)
```
Comment (vhf) [inline free-courses-en.md]: I see a "free" label on the first two chapters. Is this thing really free, or only the first two chapters?

Comment (switchhax) [inline free-courses-en.md]: Nope, i don´t think. There´s a [page](https://www.codeschool.com/pricing) with pricing plans. So, i guess, it´s not completely free.

**Pull request closed without merging**